terraform-modules/k8s/k8s-cluster.tf

```diff
@@ -59,36 +55,37 @@ resource "google_container_cluster" "cluster" {
   # CIS compliance: Enable Alias IP Ranges. According to the terrafform
   # docs, setting these values blank gets default-size ranges automatically
   # chosen: https://www.terraform.io/docs/providers/google/r/container_cluster.html#ip_allocation_policy
-  ip_allocation_policy = ["${var.ip_allocation_policy}"]
+  ip_allocation_policy = var.ip_allocation_policy
 
   # CIS compliance: Enable PodSecurityPolicyController
   pod_security_policy_config {
     enabled = true
   }
 
-  # OMISSION: CIS compliance: Enable Private Cluster 
-  private_cluster_config = ["${var.private_cluster_config}"]
+  # OMISSION: CIS compliance: Enable Private Cluster
+  private_cluster_config {
+    enable_private_endpoint = var.enable_private_endpoint
+    enable_private_nodes = var.enable_private_nodes
+    master_ipv4_cidr_block = var.private_master_ipv4_cidr_block
+  }
 
   master_authorized_networks_config {
-    cidr_blocks = ["${var.master_authorized_network_cidrs}"]
+    dynamic "cidr_blocks" {
+      for_each = var.master_authorized_network_cidrs
+      content {
+        cidr_block = cidr_blocks.value
+      }
+    }
   }
 
   addons_config {
     kubernetes_dashboard {
       # CIS compliance: Disable dashboard
-      disabled    = "true"
+      disabled = true
     }
 
     network_policy_config {
       disabled = false
     }
   }
 }
-
-output "cluster_name" {
-  value = "${google_container_cluster.cluster.name}"
-}
-
-output "cluster_endpoint" {
-  value = "${google_container_cluster.cluster.endpoint}"
-}
```

Comment:
I moved these to a separate `outputs.tf` file to match the TF recommended conventions, but I can move them back if others want.

Reply:
thanks!